section2/exercises/if_statements.rb

```diff
@@ -61,5 +64,21 @@
 # Experiment with manipulating the value held within both variables
 # to make sure all above conditions output what you expect.
 
-cups_of_flour = 1
+cups_of_flour = 3
 has_sauce = true
+
+if cups_of_flour == 1 and has_sauce == true
+  p "I cannot make pizza"
+end
+
+if cups_of_flour == 5 and has_sauce == false
```

Comment:
In Ruby, we tend to use `&&` instead of the `and` keyword. There are some differences that you can look up on your own!

But rule of thumb for Turing is that we will always expect you to use the `&&` operator.